features/step_definitions/smokey_steps.rb

```diff
@@ -135,9 +135,9 @@ def should_see(text)
 
 Then /^I should see "(.*)"$/ do |content|
   if @response
-    @response.body.include?(content).should == true
+    expect(@response.body).to include(content)
   elsif page
-    page.body.include?(content).should == true
+    expect(page.body).to include(content)
```

Comment:
@alexmuller I was getting deprecation warnings when running my tests. We should go through and replace everything at some point.